server/cherry_pick.go

```diff
@@ -119,7 +119,7 @@ func (s *Server) doCherryPick(ctx context.Context, version string, milestoneNumb
 	)
 	out, err := cmd.Output()
 	if err != nil {
-		mlog.Error("cmd.Run() failed", mlog.Err(err), mlog.String("cmdOut", string(out)))
+		mlog.Error("cmd.Run() failed", mlog.Err(err), mlog.String("cmdOut", string(out)), mlog.String("repo", pr.RepoName), mlog.Int("PR", pr.Number))
```

Comment:
Nit: would prefer the mlog.XXX fields to be in new lines.

Reply:
Yep, agreed. Reformatted for better readability